### ISSUE TITLE
OpenBSD: ed.cpp 'EPROTO' was not declared in this scope

### ISSUE THREAD

**Issue by dt1973** [eventmachine/eventmachine#697]
Similar to https://github.com/eventmachine/eventmachine/issues/676 but for OpenBSD 5.7:

```
% sudo gem install eventmachine -v '1.0.9.1'
Fetching: eventmachine-1.0.9.1.gem (100%)
Building native extensions.  This could take a while...
ERROR:  Error installing eventmachine:
        ERROR: Failed to build gem native extension.

    /usr/local/bin/ruby22 -r ./siteconf20160314-28421-1kaqomp.rb extconf.rb
checking for main() in -lcrypto... yes
checking for main() in -lssl... yes
checking for openssl/ssl.h... yes
checking for openssl/err.h... yes
checking for rb_trap_immediate in ruby.h,rubysig.h... no
checking for rb_thread_blocking_region()... no
checking for ruby/thread.h... yes
checking for rb_thread_call_without_gvl() in ruby/thread.h... yes
checking for inotify_init() in sys/inotify.h... no
checking for __NR_inotify_init in sys/syscall.h... no
checking for writev() in sys/uio.h... yes
checking for rb_thread_fd_select()... yes
checking for rb_fdset_t in ruby/intern.h... yes
checking for pipe2() in unistd.h... yes
checking for accept4() in sys/socket.h... yes
checking for SOCK_CLOEXEC in sys/socket.h... yes
checking for rb_wait_for_single_fd()... yes
checking for rb_enable_interrupt()... no
checking for rb_time_new()... yes
checking for sys/event.h... yes
checking for sys/queue.h... yes
checking for clock_gettime()... yes
checking for CLOCK_MONOTONIC_RAW in time.h... no
checking for CLOCK_MONOTONIC in time.h... yes
CXXFLAGS=-O2 -pipe -Wall -Wextra -Wno-deprecated-declarations -Wno-address
creating Makefile

make "DESTDIR=" clean

make "DESTDIR="
compiling binder.cpp
compiling cmain.cpp
compiling ed.cpp
ed.cpp: In member function 'void ConnectionDescriptor::_DispatchInboundData(const char*, long unsigned int)':
ed.cpp:877: error: 'EPROTO' was not declared in this scope
*** Error 1 in /usr/local/lib/ruby/gems/2.2/gems/eventmachine-1.0.9.1/ext (Makefile:230 'ed.o': @ c++ -I. -I/usr/local/include/ruby-2.2/i386...)

make failed, exit code 1

Gem files will remain installed in /usr/local/lib/ruby/gems/2.2/gems/eventmachine-1.0.9.1 for inspection.
Results logged to /usr/local/lib/ruby/gems/2.2/extensions/x86-openbsd/2.2/eventmachine-1.0.9.1/gem_make.out
```

Thanks!

**Comment by sodabrew** [comment]
Same issue already resolved by #690. Will be in the 1.2.0 release shortly.

**Comment by dt1973** [comment]
Much obliged @sodabrew!

cc @jeremyevans
